quality/finder/finder_hyper.inst.cfg

```diff
@@ -0,0 +1,57 @@
+[general]
+version = 4
+name = Standard Resolution
+definition = finder
+
+[metadata]
+setting_version = 13
+type = quality
+quality_type = hyper
+weight = 0
+global_quality = True
+
+[values]
+layer_height = 0.08
+layer_height_0 = 0.20
```

Comment:
i would print my first layer height at  0.1mm else i see there will be elephant foot.

Reply:
Makes sense, I'll update the pull request with that.